### PR TITLE
Ch refactor helper folder ds #170844806

### PR DESCRIPTION
**Pull request by Muhire-Josue**
#### What does this PR do?

- It reduces files in the helpers' folder by creating three of classes

#### Description of Task to be completed?

- It creates the response class to  handle the responses 

- it creates the AnnouncementQuery and UserQuery to query both models

- it validates the id passed in the parameter

- it validates the routes

- it adds more tests

#### How should this be manually tested?

- Use this link in postman: https://anounce-it.herokuapp.com/api/v1

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [170844806](https://www.pivotaltracker.com/story/show/170844806)

#### Screenshots (if appropriate)
incorret-routes
invalid-id

### CODE REVIEW

File: server/Tests/data/mockData.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ const mockData = [
     end_date: '2020-03-19',
   },
   {
-    text: 'Hello world',
-    end_date: '12-12-1998',
+    text: 'Hello world, I am John Doe',
+    end_date: '2020-03-19-0',
```
Comment (j4l13n) [inline server/Tests/data/mockData.js:42]: The date is not valid here

File: .travis.yml [hunk 2]
```diff
@@ -6,4 +6,4 @@ install: npm install
 
 script: npm test
 
-after_script: npm run coveralls
\ No newline at end of file
+after_script: npm run coverage
```
Comment (j4l13n) [inline .travis.yml]: Fix this line issue

**Review by j4l13n — COMMENTED** [review]
You have done a good job @Muhire-Josue, but consider the feedback I have added in your files before you merge this PR

File: .travis.yml (hunk 2 above)
Comment (Muhire-Josue) [reply .travis.yml]: Fixed.

**Review by j4l13n — APPROVED** [review]

**Pull request merged**